cmd/openshift-tests/e2e.go

```diff
@@ -821,3 +825,113 @@ func getEventsByPod(events []*monitor.EventInterval) map[string][]*monitor.Event
 	}
 	return eventsByPods
 }
+
+func testOperatorStateTransitions(events []*monitor.EventInterval) []*ginkgo.JUnitTestCase {
+	ret := []*ginkgo.JUnitTestCase{}
+
+	knownOperators := allOperators(events)
+	eventsByOperator := getEventsByOperator(events)
+	for _, condition := range []string{"Available", "Degraded", "Progressing"} {
+		for _, operatorName := range knownOperators.List() {
+			bzComponent := synthetictests.GetBugzillaComponentForOperator(operatorName)
+			if bzComponent == "Unknown" {
+				bzComponent = operatorName
+			}
+			testName := fmt.Sprintf("[bz-%v] clusteroperator/%v should not change condition/%v", bzComponent, operatorName, condition)
```

Comment:
would it help to clarify this further like "... should not change condition after installation is complete" ?